dss_datamover/utils/config.py

```diff
@@ -57,7 +57,10 @@ def process_config(self):
             config = json.loads(cfg.read().decode('UTF-8', "ignore"))
         if self.params:
             for param in self.params:
-                config[param] = self.params[param]
+                if self.params[param] is not None:  # must explicitly check for None since we still want False values to be overriden onto the config dict
+                    config[param] = self.params[param]
+        # process compaction options accordingly
+        config['compaction'] = True if 'compaction' not in config else config['compaction'] == 'yes'
```

Comment:
Shouldn't this be 
```        
config['compaction'] = True if 'compaction' not in config else config['compaction'] == False
```

Reply:
no because we are comparing against user input yes/no. I tested this in a python shell and it works to always resolve to a True/False. I also tested all 4 scenarios again with this change.